src/usePerpsMetadata.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from '@tanstack/react-query';
+import { ethers } from 'ethers';
+import { useImportContract } from './useImports';
+import { useSynthetix } from './useSynthetix';
+
+interface PerpsMetadata {
+  name: string;
+  symbol: string;
+}
+
```

Comment:
```suggestion
interface PerpsMetadata {
  name: string;
  symbol: string;
}

```

Rely on simple types and not introduce any new types (as much as possible)

Reply:
Removed interface. Used simple object.